drivers/oled/ssd1306_sh1106.c

```diff
@@ -179,12 +179,14 @@ bool oled_init(oled_rotation_t rotation) {
         I2C_CMD,
         DISPLAY_OFF,
         DISPLAY_CLOCK,
-        0x80,
+        OLED_DISPLAY_CLOCK,
         MULTIPLEX_RATIO,
         OLED_DISPLAY_HEIGHT - 1,
         DISPLAY_OFFSET,
         0x00,
         DISPLAY_START_LINE | 0x00,
+        VCOM_DETECT,
+        OLED_VCOM_DETECT,
```

Comment:
```suggestion
```

This is set (again) a few lines lower (L215, or 217), This change should be moved to there. Otherwise, it just gets overwritten.

Reply:
thanks, yes, my mistake. I'll just remove this for now, pending further hardware tests.